client/go/internal/cli/cmd/cert.go

```diff
@@ -182,7 +183,11 @@ func requireCertificate(force, ignoreZip bool, cli *CLI, target vespa.Target, pk
 	if force {
 		return copyCertificate(tlsOptions, cli, pkg)
 	}
-	if pkg.HasCertificate() {
+	servicesXML, err := readServicesXML(pkg)
+	if err != nil {
+		return fmt.Errorf("Error reading services.xml: %w", err)
```

Comment:
Nit: Error messages should be lower-case as this may be concatenated with other errors further up.

Reply:
Ah good spot!